tests/run_tests.py

```diff
@@ -86,13 +87,20 @@ def _initialize_coverage():
     return cov
 
 
-def _finalize_coverage(cov):
+def _finalize_coverage(cov, is_html_coverage):
     """
     Stops covering code and writes out coverage.xml in the current directory.
     """
     cov.stop()
-    cov.report()
-    cov.xml_report(outfile="coverage.xml")
+
```

Comment:
`--with-html-coverage` now allows to view the results directly in a web browser.